src/main/java/org/reaktivity/nukleus/http/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -417,7 +476,10 @@ private void processReset(
             int length)
         {
             resetRO.wrap(buffer, index, index + length);
-
+            if (slotIndex != SLOT_NOT_AVAILABLE)
+            {
+                slab.release(slotIndex);
+            }
```

Comment:
done and changed all call sites. Renamed SLOT_NOT_AVAILABLE to NO_SLOT so it makes more sense.